app/models/Answer.py

```diff
@@ -27,6 +30,15 @@ class Answer(db.Model):
     user = db.relationship('User', backref='answers')
     post = db.relationship('Post', backref='answers')
 
+    @hybrid_property
```

Comment:
Why is this decorator necessary?

Reply:
SQLAlchemy docs say I need it with the byte_len.expression also it makes it act like a getter